operator/README.md

```diff
@@ -0,0 +1,25 @@
+## Deployment
+
+The operator is deployed only in `Global-Continuous`. 
+
+### Instructions
+
+1. Set AWS profile to `incognia-production-continuous`
+
+1. Set kubernetes context to `arn:aws:eks:us-east-1:779099367007:cluster/Global-Continuous`
+
+1. Set kubernetes namespace to `kube-actions`
+
+1. Update `operator/k8s/production/kustomization.yaml` with the Github commit to be deployed.
+
+1. Generate kubernetes manifests
+
+```shell
+kustomize build --enable-alpha-plugins k8s/production
+```
+
+1. Apply changes in cluster
```

Comment:
All the steps are using 1.
Is that correct?

Reply:
Yes, that's a markdown thing to help writing.

![image](https://github.com/inloco/kube-actions/assets/23252082/c133d6fc-afd3-4d1b-b766-91bc75993c61)